assets/terraform/aws/bootstrap.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# File passed to VM at creation time
+#
+apt update 
+apt install -y python-pip lvm2
+pip install --upgrade awscli
+
+set -euo pipefail
+
+pvcreate -ff /dev/xvde
+vgcreate data /dev/xvde
+lvcreate -n var_lib_gravity -l 20%VG data
+lvcreate -n var_lib_data -l 20%VG data
+lvcreate -n etcd -l 60%VG data
```

Comment:
>> Can't use TF defaults?
Yes, TF would be just provisioning templates; all configuration values would come top-down.